src/GitHub.Exports.Reactive/Services/IRepositoryCloneService.cs

```diff
@@ -23,7 +23,7 @@ public interface IRepositoryCloneService
         /// <param name="repositoryPath">The directory that will contain the repository directory.</param>
         /// <param name="progress">
         /// An object through which to report progress. This must be of type
-        /// <see cref="System.IProgress{Microsoft.VisualStudio.Shell.ServiceProgressData}"/>, but
+        /// System.IProgress&lt;Microsoft.VisualStudio.Shell.ServiceProgressData&gt;, but
```

Comment:
What happened here?

Reply:
It was an incorrect `<see>` tag. The "cref" value could provide a link to `System.IProgress{T}` or `Microsoft.VisualStudio.Shell.ServiceProgressData`, but there is no link to documentation for `System.IProgress{Microsoft.VisualStudio.Shell.ServiceProgressData}`.

Reply:
asdf

Reply:
asdf

Reply:
asdf